modules/cluster-logging-deploy-multitenant.adoc

```diff
@@ -0,0 +1,43 @@
+:_module-type: PROCEDURE
+// Module included in the following assemblies:
+//
+// * assemblies/cluster-logging-deploying.adoc
+
+[id="cluster-logging-deploy-multitenant_{context}"]
+= Installing cluster logging into a multitenant network
+
+[role="_abstract"]
+If you are deploying cluster logging into a cluster that uses multitenant isolation mode, projects are isolated from other projects. As a result, network traffic is not allowed between pods or services in different projects.
+
+Because the Elasticsearch Operator and the Cluster Logging Operator are installed in different projects, you must explicitly allow access between the  `openshift-operators-redhat` and `openshift-logging` projects. How you allow this access depends on how you configured multitenant isolation mode.
+
+.Procedure
+
+To allow traffic between the Elasticsearch Operator and the Cluster Logging Operator, perform one of the following:
+
+* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *Multitenant* mode, use the following command to join the two projects:
++
+For example:
++
+[source,terminal]
+----
+$ oc adm pod-network join-projects --to=openshift-operators-redhat openshift-logging
+----
+
+* If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *NetworkPolicy* mode, create a network policy object in the `openshift-logging` namespace that allows ingress from the `openshift-operators-redhat` project to the `openshift-logging` project.
++
+For example:
++
+[source,yaml]
+----
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-openshift-operators-redhat
+spec:
+  ingress:
+    - from:
+      - namespaceSelector:
+          matchLabels:
+            project: openshift-operators-redhat
```

Comment:
There isn't such label in project  openshift-operators-redhat. I think we should use label 'name: openshift-operators-redhat'

Reply:
I think we need to expose the router to monitoring too.
{
    "apiVersion": "networking.k8s.io/v1",
    "kind": "NetworkPolicy",
    "metadata": {
        "name": "allow-from-openshift-monitoring",
        "namespace": "openshift-logging"
    },
    "spec": {
        "ingress": [
            {
                "from": [
                    {
                        "namespaceSelector": {
                            "matchLabels": {
                                "network.openshift.io/policy-group": "monitoring"
                            }
                        }
                    }
                ]
            }
        ],
        "podSelector": {},
        "policyTypes": [
            "Ingress"
        ]
    }
}

Reply:
@anpingli @jeremyeder  I have updated this topic, but please check my YAML formatting. I do not know how to interpret {} and [

Thanks.

Reply:
@anpingli where is that 2nd network policy requirement coming from?  I don't see it in the OCP docs.

Reply:
Based on a conversation with Jeremy, I have removed the 2nd network policy from the docs for now.

I have fixed the first policy by changing the label to: 'name: openshift-operators-redhat'

Reply:
@zhaozhanqi shall we create NetworkPolicy/allow-from-openshift-monitoring in If you configured multitenant isolation mode with the OpenShift SDN CNI plug-in set to the *NetworkPolicy* ?

Reply:
@jonquilwilliams  The allow-from-openshift-monitoring should have been created when enabling networkpolicy-multitenant-isolation.
  https://docs.openshift.com/container-platform/4.1/networking/configuring-networkpolicy.html#nw-networkpolicy-multitenant-isolation_configuring-networkpolicy-plugin

Reply:
This is not exposing a router.
The network policy listed here is to allow prometheus to scrape pods in `openshift-loggin` as you said @jeremyeder .
However, this only applies to OCP clusters with **networkpolicy-multitenant-isolation enabled**, which is not the case for OSD.

Reply:
Please see my latest comments below. This should be resolved now because Sammi is updating the Network Policy chapter, which is a separate PR.